gorp_test.go

```diff
@@ -831,6 +831,37 @@ func TestSetUniqueTogether(t *testing.T) {
 	}
 }
 
+func TestSetUniqueTogetherIdempotent(t *testing.T) {
+	dbmap := newDbMap()
+	table := dbmap.AddTable(UniqueColumns{}).SetUniqueTogether("FirstName", "LastName")
+	table.SetUniqueTogether("FirstName", "LastName")
+	table.SetUniqueTogether("LastName", "FirstName")
```

Comment:
I think it can work for us, but I'm in the SQL semantics an Index (unique or not) A,B is not the same than B,A. Actually we can be introducing bugs in other places where that unique index are been used in an specific order.

Reply:
I agree that these would be different indexes, with different usage semantics. In our case, it seems like we're only using the "unique" part of this given that we then go to the effort of creating other indexes for the columns in question, but agree that this could be confusing. (Another reason to manage the SQL ourselves, IMO!)

We don't need the out-of-order handling, strictly speaking, so I'll remove it to avoid future confusion.